src/test/java/pages/BasePage.java

```diff
@@ -0,0 +1,46 @@
+package pages;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import services.WaitService;
+
+import java.awt.*;
+import java.awt.datatransfer.Clipboard;
+import java.awt.datatransfer.StringSelection;
+
+import static tests.BaseTest.getDriver;
+
+public abstract class BasePage {
+
+    WebDriver driver;
+    WaitService waitService;
+    Actions builder;
+    JavascriptExecutor executor;
+
+    public BasePage() {
+        this.driver = getDriver();
```

Comment:
Why do you use 'this' for driver variable but not for others?

Reply:
fixed